Runtime/Scripts/BaseMeshSync.cs

```diff
@@ -1746,13 +1746,110 @@ private static bool HandleKeywords(Material destMat, string keyword)
         return rec;
     }
 
+    static Transform FindOrCreateByPath(Transform parent, string path, Action<string> parentCreationCallback, bool worldPositionStays = true) {
+        string[] names = path.Split('/');
+        if (names.Length <= 0)
+            return null;
+
+        static Transform FindFirstRoot(string objectName) {
```

Comment:
FindFirstRoot seems to be the same with the one in FilmInternalUtilities ?
So we can use `FilmInternalUtilities.GameObjectUtility.FindFirstRoot(rootGameObjectName);`

Reply:
Right, I forgot we had access to the internal methods.